website/docs/cloud-docs/api-docs/state-versions.mdx

```diff
@@ -608,6 +608,125 @@ curl \
+
+### Request Body
+
+This PATCH endpoint requires a JSON object with the following properties as a request payload.
+
+Properties without a default value are required.
+
+| Key path                                            | Type   | Default | Description                                                  |
+|-----------------------------------------------------|--------|---------|--------------------------------------------------------------|
+| `data.type`                                         | string |         | Must be `"state-versions"`.                                  |
+| `data.relationships.reverted-state-version.data.id` | string |         | The ID of the state version to use for the revert operation. |
+
+### Sample Payload
+
+```json
+{
+  "data": {
+    "type":"state-versions",
+    "relationships": {
+      "reverted-state-version": {
+        "data": {
+          "type": "state-versions",
+          "id": "sv-bWfq4Y1YpRKW4mx7"
+        }
+      }
+    }
+  }
+}
+```
+
+### Sample Request
+
+```shell
+curl \
+  --header "Authorization: Bearer $TOKEN" \
+  --header "Content-Type: application/vnd.api+json" \
+  --request PATCH \
+  --data @payload.json \
+  https://app.terraform.io/api/v2/workspaces/ws-6fHMCom98SDXSQUv/state-versions
+```
+
+### Sample Response
+
+```json
+{
+    "data": {
+        "id": "sv-DmoXecHePnNznaA4",
+        "type": "state-versions",
+        "attributes": {
+            "created-at": "2022-11-22T01:22:03.794Z",
+            "size": 940,
+            "hosted-state-download-url": "https://archivist.terraform.io/v1/object/...",
+            "hosted-json-state-download-url": "https://archivist.terraform.io/v1/object/...",
+            "modules": {
+                "root": {
+                    "null-resource": 1,
+                    "data.terraform-remote-state": 1
+                }
+            },
+            "providers": {
+                "provider[\"terraform.io/builtin/terraform\"]": {
+                    "data.terraform-remote-state": 1
+                },
+                "provider[\"registry.terraform.io/hashicorp/null\"]": {
+                    "null-resource": 1
+                }
+            },
+            "resources": [
+                {
+                    "name": "other_username",
+                    "type": "data.terraform_remote_state",
+                    "count": 1,
+                    "module": "root",
+                    "provider": "provider[\"terraform.io/builtin/terraform\"]"
+                },
+                {
+                    "name": "random",
+                    "type": "null_resource",
+                    "count": 1,
+                    "module": "root",
+                    "provider": "provider[\"registry.terraform.io/hashicorp/null\"]"
+                }
+            ],
+            "resources-processed": false,
+            "serial": 9,
+            "state-version": 4,
+            "terraform-version": "1.3.5",
+        },
+        "links": {
```

Comment:
To update the response body to carry the new reverted state relation and other existing relationship as the show API:
```
        "relationships": {
            "reverted-state-version": {
                "data": {
                    "id": "sv-YfmFLgTv31VZsP",
                    "type": "state-versions"
                }
            },
```